### PR TITLE
added Avatar Select Service

### PR DESCRIPTION
**Pull request by soniaigartua**
added Avatar Select Service

Evidence:

### CODE REVIEW

File: app/src/main/java/com/example/pps_tudai/activity/WelcomeActivity.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    private WelcomePresenter presenter;
+    private int userId;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_welcome);
+
+        Bundle data = getIntent().getExtras();
```
Comment (SartiMau) [inline app/src/main/java/com/example/pps_tudai/activity/WelcomeActivity.java]: this is not needed, you can use getIntent().getIntExtra()

File: app/src/main/java/com/example/pps_tudai/adapter/AvatarAdapter.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        }
+
+        @OnClick(R.id.img_avatar)
+        public void onDetailsClick(View view) {
+            new AvatarDialog(view.getContext(), avatar, userId).show();
+        }
+
+        public void bind(final AvatarAPIResponse.Result avatar) {
+            this.avatar = avatar;
+            url = avatar.getThumbnail().getPath() + DOT + avatar.getThumbnail().getExtension();
```
Comment (SartiMau) [inline app/src/main/java/com/example/pps_tudai/adapter/AvatarAdapter.java]: you have to do this in the onBinfViewHolder method

File: app/src/main/java/com/example/pps_tudai/mvp/view/WelcomeView.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    public WelcomeView(Activity activity) {
+        ButterKnife.bind(this, activity);
+        this.activityWeak = new WeakReference<Activity>(activity);
+    }
+
+    public void configSreen(User user) {
+        if (user.getUrl_image() != null) {
+            Picasso.get().load(user.getUrl_image())
+                    .resize(180,230)
```
Comment (SartiMau) [inline app/src/main/java/com/example/pps_tudai/mvp/view/WelcomeView.java]: use contansts pls, no hardcode

File: app/src/main/java/com/example/pps_tudai/activity/WelcomeActivity.java (hunk 1 above)
Comment (soniaigartua) [reply app/src/main/java/com/example/pps_tudai/activity/WelcomeActivity.java]: done!

File: app/src/main/java/com/example/pps_tudai/adapter/AvatarAdapter.java (hunk 2 above)
Comment (soniaigartua) [reply app/src/main/java/com/example/pps_tudai/adapter/AvatarAdapter.java]: done!

File: app/src/main/java/com/example/pps_tudai/mvp/view/WelcomeView.java (hunk 3 above)
Comment (soniaigartua) [reply app/src/main/java/com/example/pps_tudai/mvp/view/WelcomeView.java]: DONE!!

**Pull request merged**